src/Model/Anon.py

```diff
@@ -726,24 +668,76 @@ def anonymize(path, Datasets, FilePaths, rawdvh):
         first_dicom_object["PatientName"].repval
     )
 
-    if not file_previously_anonymised:
-        hashed_patient_id = _gen_md5_and_sha1_hash(Original_P_ID)
-        hashed_patient_name = _gen_md5_and_sha1_hash(patient_name_in_dataset)
+    # get the pname_ID ("Patient Name  + PatientID") before any anonymisation
```

Comment:
note that this change is in the public anonymize() function, the diff viewer may make this appear to be in the private _anonymise_dicom_data()